lib/analysis/network.py

```diff
@@ -625,7 +625,7 @@ def bin_increment(nick_name, messager, nick_spliced, bins, bin_index):
             bins = [0] * no_of_bins
 
             for line in day_log:
-                if(line[0] != '='): 
+                if(line[0] == '['):
```

Comment:
Will this change effect network analysis of other channels like ubuntu and slackware? If not, why not?

Reply:
1) Does not effect slackware at all because @kaushik-rohit has created a different network.py for slackware.
2) Doesn't effect ubuntu, because ubuntu logs has 2 types of messages, chat messages that starts with time stamp (`[00:00]`) and bot messages that start with `===`. So for ubuntu the if condition effectively remains the same. I also crosschecked this by comparing the results for bins before and post the change.